projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.ts

```diff
@@ -35,8 +35,7 @@ export class AddToCartComponent implements OnInit, OnDestroy {
 
   hasStock = false;
   quantity = 1;
-  increment = false;
-  cartEntry$: Observable<OrderEntry>;
```

Comment:
Removing public/protected members is a breaking change. We need to document it in the migration docs of 3.0. I created a ticket for it https://github.com/SAP/spartacus/issues/9286 with milestone 3.0